.gitignore

```diff
@@ -16,12 +16,10 @@ config/*.old
 *~
 crossgcc
 clean
-*.map
 *.sec
 *.dep
 *.ffs
 *.vol
 *.lz
 *.fv
-*.bin
```

Comment:
Not sure about this. Adding it manually for devels is one thing, but not getting "dirty" builds when modified is another? This is more for debating then refusal of that change.

For example on older boards where ifd.bin is modified but ifd is in git, the user will get a dirty build where git status would inform that ifd was changed under blobs dir. The dev/user can then add the ifd and sign to get rid of "dirty" builds or accept that the build he flashed is dirty.

Reply:
@tlaurion I'm _removing_ *.bin.  It's ignored in master now.  It doesn't affect files added to the index already AFAIK, but does affect whether dropping a new .bin file shows up as a change and whether `git add` will actually add it or silently ignore it.